analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -637,37 +638,61 @@ def main(args):
 
     report_dir = args.output_path
 
+    ctu_or_stats_enabled = False
     # Skip list is applied only in pre-analysis
-    # if --ctu-collect or --stats-collect was called explicitly.
+    # if --ctu-collect was called explicitly.
     pre_analysis_skip_handler = None
     if 'ctu_phases' in args:
         ctu_collect = args.ctu_phases[0]
         ctu_analyze = args.ctu_phases[1]
-        if ((ctu_collect and not ctu_analyze)
-                or ("stats_output" in args and args.stats_output)):
+        if ctu_collect and not ctu_analyze:
             pre_analysis_skip_handler = skip_handler
 
+        if ctu_collect or ctu_analyze:
+            ctu_or_stats_enabled = True
+
+    # Skip list is applied only in pre-analysis
+    # if --stats-collect was called explicitly.
+    if 'stats_output' in args and args.stats_output:
+        pre_analysis_skip_handler = skip_handler
+        ctu_or_stats_enabled = True
+
+    if 'stats_enabled' in args and args.stats_enabled:
+        ctu_or_stats_enabled = True
+
     context = analyzer_context.get_context()
     analyzer_env = env.extend(context.path_env_extra,
                               context.ld_lib_path_extra)
 
     # Parse the JSON CCDBs and retrieve the compile commands.
     actions = []
+
+    # Number of all the compilation commands in the parsed log files,
+    # logged by the logger.
+    all_cmp_cmd_count = 0
+    # Number of the skipped compile commands during the
+    # compile command processing.
+    skipped_cmp_cmd_count = 0
+
     for log_file in args.logfile:
         if not os.path.exists(log_file):
             LOG.error("The specified logfile '%s' does not exist!",
                       log_file)
             continue
-
-        actions += log_parser.parse_unique_log(
-            load_json_or_empty(log_file),
+        compile_commands = load_json_or_empty(log_file)
```

Comment:
Set the default value:
```suggestion
        compile_commands = load_json_or_empty(log_file, {})
```